cni/cmd/istio-cni-repair/main.go

```diff
@@ -180,15 +196,20 @@ func main() {
 		log.Fatalf("Could not construct clientSet: %s", err)
 	}
 
-	podFixer := repair.NewBrokenPodReconciler(clientSet, filters, options.RepairOptions)
+	podFixer := repair.NewBrokenPodReconciler(clientSet, filters, options.RepairOptions, &metrics)
 	logCurrentOptions(&podFixer, options)
+	stopCh := make(chan struct{})
```

Comment:
stopCh is never closed? Maybe we need SIGTERM handler. Can be a followup - it already had this issue before this PR